content/applications/hr/fleet/accidents.rst

```diff
@@ -0,0 +1,257 @@
+=========
+Accidents
+=========
+
+When managing a fleet of vehicles, eventually, accidents will occur. It is important to track these
+occurrences to gain insights, from seeing how much vehicles cost to keep on the road, to identifying
+the safest drivers.
+
+Due to the customizable nature of Odoo's *Fleet* app, there are several different ways to track
+accidents. The instructions below provide step by step instructions for only **one** method to keep
+track of accidents and their subsequent repair costs.
+
+Structure
+=========
+
+For this example, to track accidents, two :ref:`service types <fleet/new-type>` are created:
+`Accident - Driver's Fault` and `Accident - No Fault`.
+
+This will make it possible to track the various repairs associated with accidents, organized by the
+type of accident (who was at fault).
+
+When an accident occurs, a service record is created. The specific repairs needed for the accident
+are logged in the *description* of the service record, and the details about the accident (what
+happened) are logged in the *notes* section.
+
+With this organizational structure, it is possible to view all accidents organized by fault, car,
+driver, or cost.
+
+.. seealso::
+   To manage accidents, the creation of service records is **required**.
+
+   Refer to the :doc:`service` documentation for detailed instructions on creating service records
+   in Odoo's *Fleet* app.
+
+Log accidents and repairs
+=========================
+
+After an accident occurs, repairs are usually needed. To log an accident, the first step is to
+:ref:`create a service record <fleet/service-form>` detailing the specific repairs needed.
+
+.. note::
+   Some accidents require multiple repairs with several different vendors. For these scenarios, a
+   separate service record is needed for each vendor performing repairs. To keep records organized,
+   it is recommended to keep the :guilabel:`NOTES` field identical, as well as attaching the same
```

Comment:
This is a super super personal and completely just my opinion nitpick, but does it *have* to be all caps? I feel like the documentation is yelling at me :sweat_smile:

Reply:
Lol- no, I don't like all caps, but it was shown that way, which is why I wrote it like that. I personally refer "Notes" but was worried it would be corrected since it's not how it appears on the UI. But since you also agree, I'll change it!